src/main/java/seedu/rc4hdb/logic/commands/modelcommands/FilterCommand.java

```diff
@@ -40,14 +38,12 @@ public class FilterCommand extends ModelCommand {
             + PREFIX_PHONE + "91234567 "
             + PREFIX_EMAIL + "johndoe@example.com";
 
-    public static final String MESSAGE_FILTER_PERSON_SUCCESS = "filtered Person: %1$s";
     public static final String MESSAGE_NOT_FILTERED = "At least one field to filter must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the resident book.";
 
     private final ResidentDescriptor filterPersonDescriptor;
 
     /**
-     * @param filterPersonDescriptor details to filter the person with
+     * @param filterPersonDescriptor description to filter the resident with
```

Comment:
I think its better for this comment to be moved to right above the `filterPersonDescriptor` attribute.